scheduler/local_scd_server.service

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 SuperDARN Canada, University of Saskatchewan
```

Comment:
There might be some copy-paste errors in the docstring between this file and the .daemon file - there's a few lines missing from this docstring for reasons unknown

Reply:
Updated